aws/policy/compute.yaml

```
@@ -29,6 +29,14 @@ Statement:
       - autoscaling:CancelInstanceRefresh
       - autoscaling:StartInstanceRefresh
       - autoscaling:UpdateAutoScalingGroup
+      - autoscaling:DescribeAutoScalingGroups
+      - autoscaling:DisableMetricsCollection
+      - autoscaling:DescribeLaunchConfigurations
+      - autoscaling:DescribeLifecycleHooks
+      - autoscaling:DetachLoadBalancerTargetGroups
+      - autoscaling:DetachLoadBalancers
+      - autoscaling:EnableMetricsCollection
+      - autoscaling:TerminateInstanceInAutoScalingGroup
```

```suggestion
      - autoscaling:DisableMetricsCollection
      - autoscaling:DetachLoadBalancerTargetGroups
      - autoscaling:DetachLoadBalancers
      - autoscaling:EnableMetricsCollection
      - autoscaling:TerminateInstanceInAutoScalingGroup
```

We have `autoscaling:Describe*` (line 116/120)